src/registrar/management/commands/utility/extra_transition_domain_helper.py

```diff
@@ -751,6 +751,257 @@ def _infer_filename(self, regex: re.Pattern, matched_file_name, default_file_nam
+            An example would be `EnumFilenames.DOMAIN_ADHOC`.
+
+            desired_id: str -> Which id you want to search on.
+            An example would be `"12"` or `"igorville.gov"`
+
+        Explanation:
+            Each data file has an associated type (file_type) for tracking purposes.
+
+            Each file_type is a dictionary which
+            contains a dictionary of row[id_field]: object.
+
+            In practice, this would look like:
+
+            EnumFilenames.AUTHORITY_ADHOC: {
+                "1": AuthorityAdhoc(...),
+                "2": AuthorityAdhoc(...),
+                ...
+            }
+
+            desired_id will then specify which id to grab. If we wanted "1",
+            then this function will return the value of id "1".
+            So, `AuthorityAdhoc(...)`
+        """
+        # Grabs a dict associated with the file_type.
+        # For example, EnumFilenames.DOMAIN_ADDITIONAL.
+        desired_type = self.parsed_data.file_data.get(file_type)
+        if desired_type is None:
+            self.parse_logs.create_log_item(
+                file_type,
+                LogCode.ERROR,
+                f"Type {file_type} does not exist",
+            )
+            return None
+
+        # Grab the value given an Id within that file_type dict.
+        # For example, "igorville.gov".
+        obj = desired_type.data.get(desired_id)
+        if obj is None:
+            self.parse_logs.create_log_item(
+                file_type,
+                LogCode.ERROR,
+                f"Id {desired_id} does not exist for {file_type.value[0]}",
+            )
+        return obj
+
+    def log_add_or_changed_values(self, file_type, values_to_check, domain_name):
+        for field_name, value in values_to_check:
+            str_exists = value is not None and value.strip() != ""
+            # Logs if we either added to this property,
+            # or modified it.
+            self._add_or_change_message(
+                file_type,
+                field_name,
+                value,
+                domain_name,
+                str_exists,
+            )
+
+    def _add_or_change_message(self, file_type, var_name, changed_value, domain_name, is_update=False):
+        """Creates a log instance when a property
+        is successfully changed on a given TransitionDomain."""
+        if not is_update:
+            self.parse_logs.create_log_item(
+                file_type,
+                LogCode.INFO,
+                f"Added {var_name} as '{changed_value}' on {domain_name}",
+                domain_name,
+                not self.debug,
+            )
+        else:
+            self.parse_logs.create_log_item(
+                file_type,
+                LogCode.WARNING,
+                f"Updated existing {var_name} to '{changed_value}' on {domain_name}",
+                domain_name,
+                not self.debug,
+            )
+
```

Comment:
All of this is just boilerplate copied from the old code. Really this logic should be moved into a class if we wanted to use it again, but given where we are at - its not really as important for the PR. Its a little overkill for just parsing two files (as opposed to the many of the other script), but it saved time by using what already exists and it is rather efficient. 

This portion of the code may be confusing to follow without context to the last PR, let me know if it is.